pool/acceptedwork.go

```diff
@@ -208,3 +208,42 @@ func ListMinedWork(db *bolt.DB) ([]*AcceptedWork, error) {
 	}
 	return minedWork, nil
 }
+
+// FetchUnconfirmedWork returns all work which is not confirmed as mined with
+// height less than the provided height.
+func FetchUnconfirmedWork(db *bolt.DB, height uint32) ([]*AcceptedWork, error) {
```

Comment:
Let's unexport the new helpers, I do not see any other call sites for them outside of the pool package currently.